conf/log4j2.yaml

```diff
@@ -134,7 +134,6 @@ Configuration:
       additivity: true
       AppenderRef:
         - ref: "${sys:pulsar.log.appender}"
-          level: "${sys:pulsar.log.level}"
```

Comment:
This should not be changed as we should keep default behavior.

Reply:
as default `pulsar.log.level=info`
i have do a test in product env as follow:
<img width="1022" alt="image" src="https://user-images.githubusercontent.com/16517186/154014948-2d2547f2-7f1b-4943-a0e2-a8096d6abd1a.png">

remove this item seems do not change default log4j behavior.

as answer in [[stackoverflow:programmatically-change-log-level-in-log4j2](https://stackoverflow.com/questions/23434252/programmatically-change-log-level-in-log4j2) :

i have try all mentioned way to change logger level by programmatically way, it not work unless remove `level: "${sys:pulsar.log.level}"`

Have you ever had similar problems?

Reply:
User may already have set `pulsar.log.level=warn` in their deployment. Will this PR change the settings?

Reply:
i have set  `pulsar.log.level=warn` for test. but not what i expected. described as this issue #14298

Reply:
> User may already have set `pulsar.log.level=warn` in their deployment. Will this PR change the settings?

i have a test 

if User set `pulsar.log.level=warn` in their deployment while starting service.

this PR will not change the settings , all loggers's level is `warn`, meanwhile , the PR featrue also works.

> ~~level: "${sys:pulsar.log.level}"~~

this line config is redundant and will disturb the programmatically way.

Reply:
> this PR will not change the settings

I am a little confused that if you removed this, how would user's previous setting (set by ENV property `PULSAR_LOG_LEVEL` in pulsar_env.sh) take effect?

Reply:
i have a test in standalone mode after removed this line. 
1. `pulsar.log.level` set by ENV property PULSAR_LOG_LEVEL in pulsar_env.sh to `warn`
<img width="224" alt="image" src="https://user-images.githubusercontent.com/16517186/155252415-39f73dfd-4d33-466b-872c-3fe71fb5e395.png">
2. then start pulsar service in standalone mode. the service log level is warn, worked as desired. 
<img width="1066" alt="image" src="https://user-images.githubusercontent.com/16517186/155252639-87e46c42-0106-4ca1-9de4-4c6168b23c9c.png">
3. then run cmd `bin/pulsar-admin brokers update-logger-level --classname org.apache.zookeeper.server.FinalRequestProcessor --level debug` to validation feature in this PR. workd as desired.
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/16517186/155252852-e9b7ccb1-740c-419f-9f67-4fd8f2072ae9.png">

so remove this line would not invalidate user's previous setting (set by ENV property `PULSAR_LOG_LEVEL` in pulsar_env.sh

Reply:
Can you explain from the code, why `pulsar.log.level` this system env could still take effect after we delete from log4j2.yaml? Did we pass this variable to log4j somewhere else?

Reply:
> Can you explain from the code, why pulsar.log.level this system env could still take effect after we delete from log4j2.yaml? Did we pass this variable to log4j somewhere else?

+1, from the source code here https://github.com/apache/pulsar/blob/master/bin/pulsar#L299, I think we can't remove this line here.